13.technical-support/04.Reporting-Tool/docs.en.md

```diff
@@ -0,0 +1,169 @@
+
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/Techsupp/forma5en.png" width="500"/>
+
+6. Provide additional information that will help filter developers understand the situation
+Also enter your account on **GitHub**, if available;
+
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/Techsupp/forma6en.png" width="500"/>
+
+7. Check each step before sending the form;
+
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/Techsupp/forma7en.png" width="500"/>
+
+8. You'll be provided with a link to track progress on your task.
+
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/Techsupp/forma8en.png" width="500"/>
+
+## Diagnosis of a issue
+
+<a id="solo"></a>
+
+This option is suitable for those who aren’t afraid of difficulties and don’t mind trying to find the problem on their own. This does not require any special skills or in-depth knowledge of HTML and CSS.
+
+For example, you are faced with a situation that your favorite website won’t open, authorization has broken, or some page elements are displayed incorrectly. The cause may be using a specific filter/extension/Stealth Mode setting. To diagnose these problems, you need to do the following:
+
+### **Filters**
+
+1. Disable **Ad Blocker**;
+2. If it solved the problem, re-enable the module and disable filters one by one. This is done to identify filters that "break" the page;
+3. If you have managed to find out which filter "breaks" a page, you should inform the developers when filling out the form (see above). In this case, it is advisable to return the filters to their original position.
+3. If turning off all the filters hasn't produced any results, rule **@@||example.com^$document,extension** should be applied(Where **example.com** is an example site name, you need to substitute your own option);
+4. In case of failure (issue not solved), turn to section **Stealth Mode**.
+
+
+### **Stealth Mode**. 
+
+1. Disable **Stealth Mode**;
+2. If this solves the problem, turn the module on and turn off the **Stealth Mode** options one by one. This is done to detect which filters are "breaking" the page;
+3 If you are able to detect which setting "breaks" a page, you should inform the developers when filling out the form (see above). In this case, it is advisable to return the settings to their original position;
+3. In case of failure(issue not solved), turn to section **Extensions**.
+
+
+### **Extensions**
+
+1. Disable **Extensions**;
+2. If this solves the problem, turn on the module and turn off the settings **Extensions** one by one. This is done to find out which setting "breaks" the page;
+3. If you are able to detect which setting "breaks" a page, you should inform the developers when filling out the form (see above). In this case it is desirable to return the settings to their original position;
```

Comment:
If you are able to detect which setting "breaks" a page, you should inform the developers when filling out the form (see above). In this case, it is desirable to return to the initial settings;